### ISSUE TITLE
Touchable props not working

### ISSUE THREAD

**Issue by Frexuz** [Ajackster/react-native-global-props#6]
```
import {
  setCustomText,
  setCustomTouchableOpacity,
  setCustomTouchableHighlight
} from 'react-native-global-props'

const customTextProps = {
  style: {
    fontSize: 16,
    color: 'green'
  }
}

const customTouchableOpacityProps = {
  activeOpacity: 0.6
}

const customTouchableHightlightProps = {
  underlayColor: 'red'
}

export function setupGlobalStyles () {
  setCustomText(customTextProps)
  setCustomTouchableOpacity(customTouchableOpacityProps)
  setCustomTouchableHighlight(customTouchableHightlightProps)
}
```

The text styles are working fine, but not `activeOpacity` or `underlayColor`

React native 0.45.1

Any ideas?

**Comment by Ajackster** [comment]
Hey @Frexuz, thanks for the issue. It seems that applying activeOpacity or underlayColor directly to the corresponding components (TouchableOpacity/TouchableHighlight) does not work. I think this is a bug within react-native as seen in this issue. https://github.com/facebook/react-native/issues/11834

**Comment by Ajackster** [comment]
Good catch. Currently I'm not checking for default props provided by react-native. activeOpacity and underlayColor both have default props, and they're set to override you custom provided props. Working on a fix now. Thanks for the report!

**Comment by Ajackster** [comment]
This is now fixed, published on v1.1.3. Let me know if you have any more problems. :)

**Comment by Frexuz** [comment]
Works, so awesome :) <3
